source/WindowScroller/WindowScroller.js

```diff
@@ -107,6 +126,12 @@ export default class WindowScroller extends Component {
 
     this._setNextState({ scrollTop })
 
+    if (this._originalBodyPointerEvents === null) {
+      this._originalBodyPointerEvents = document.body.style.pointerEvents
+      document.body.style.pointerEvents = 'none'
```

Comment:
I wonder if this has any weird layout/repainting side effects (changing body style)?